solr/solrj/src/java/org/apache/solr/client/solrj/request/DelegationTokenRequest.java

```diff
@@ -84,8 +84,8 @@ public DelegationTokenResponse.Get createResponse(SolrClient client) {
     }
 
     @Override
-    public String getRequestType() {
-      return SolrRequestType.ADMIN.toString();
+    protected SolrRequestType getBaseRequestType() {
```

Comment:
[Q] Unrelated to this PR, but does DelegationTokenRequest serve a purpose anymore?  SOLR-9200 added it originally in order to support 'hadoop-auth', and now that the "hadoop-auth" module is gone this is only referenced in tests afaict.

Perhaps it could've been removed when @epugh removed hadoop-auth earlier this year?

Reply:
@risdenk I also at-mentioned you on that point on the related Hadoop removal PR since you are the SME on this (not Pugh).